content/_components/ebay/actions.md

```diff
@@ -13,108 +13,84 @@ ComponentVersion: 2.0.2
 
 See the detailed info about this action [here](https://developer.ebay.com/api-docs/buy/feed/resources/item/methods/getItemFeed).
 
-### Input json schema location
-
-Request JSON schema ./src/main/resources/schemas/json/ItemFeed.in.json
-Response JSON schema ./src/main/resources/schemas/json/ItemFeed.out.json
-
 ## Get Item Description Feed
 
 The Description feed file is generated each day. This call lets you download a daily TSV_GZIP (tab separated value gzip) Description feed file containing the descriptions of all the items that were listed on a specific day in a specific category.
 
-![Get Item Description Feed](img/get-item-description-feed.png)
+{% include img.html max-width="100%" url="img/get-item-description-feed.png" title="Get Item Description Feed" %}
 
 The Description feed file contains only the itemId, itemGroupId and description columns. The value of the description column is BASE64 encoded. For each row, there will be values in either itemId or itemGroupId. The description column will always contain a value.
 
 The other parameters and fields are the same like in GetItemFeed action.
 
 Documentation for this action could be found [here](https://developer.ebay.com/api-docs/buy/feed/resources/item_description/methods/getItemDescriptionFeed).
 
-### Input json schema location
-
-Request JSON schema ./src/main/resources/schemas/json/ItemFeed.in.json
-
-Response JSON schema ./src/main/resources/schemas/json/ItemFeed.out.json
-
 ## Get Item Snapshot Feed
 
 The Hourly feed file is generated each hour every day for all categories. This call lets you download an Hourly TSV_GZIP (tab separated value gzip) feed file containing all the items that have changed within the specified day and hour for a specific category. This means to generate the 8AM file of items that have changed from 8AM and 8:59AM, the service starts at 9AM. You can retrieve the 8AM snapshot file at 10AM.
 
-![Get Item Snapshot Feed](img/get-item-snapshot-feed.png)
+{% include img.html max-width="100%" url="img/get-item-snapshot-feed.png" title="Get Item Snapshot Feed" %}
 
 You can use the response from this call to update the item details of items stored in your database. By comparing the value of itemSnapshotDate for the same item you will be able to tell which information is the latest.
 
-> **Note:** Currently, hourly updates for descriptions are not supported.
-
-All the parameters and fields are the same like in GetItemFeed and GetItemDescriptionFeed action.
-The only difference is in the format of data:
+> **Please note:** Currently, hourly updates for descriptions are not supported.
 
-> **Important:** The format of the 'data' input field: UTC format (yyyy-MM-ddThh:00:00.000Z).
+All the parameters and fields are the same like in [Get Item Feed](#get-item-feed) and [Get Item Description Feed](#get-item-description-feed) action. The only **difference** is in the format of data: UTC format (yyyy-MM-ddThh:00:00.000Z). E.g.: 2017-07-12T09:00:00.000Z.
```

Comment:
All the parameters and fields are the same as in [Get Item Feed](#get-item-feed) and [Get Item Description Feed](#get-item-description-feed) actions. The only **difference** is in the format of the data: UTC format (yyyy-MM-ddThh:00:00.000Z). E.g.: 2017-07-12T09:00:00.000Z.